app/code/Magento/Checkout/Test/Unit/Model/ShippingInformationManagementTest.php

```diff
@@ -6,101 +6,161 @@
 
 namespace Magento\Checkout\Test\Unit\Model;
 
+use Magento\Checkout\Api\Data\PaymentDetailsInterface;
+use Magento\Checkout\Api\Data\ShippingInformationInterface;
+use Magento\Checkout\Model\PaymentDetailsFactory;
+use Magento\Checkout\Model\ShippingInformationManagement;
+use Magento\Framework\Exception\InputException;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Exception\StateException;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Magento\Quote\Api\CartTotalRepositoryInterface;
+use Magento\Quote\Api\Data\AddressInterface;
+use Magento\Quote\Api\Data\CartExtension;
+use Magento\Quote\Api\Data\CartExtensionFactory;
+use Magento\Quote\Api\Data\PaymentMethodInterface;
+use Magento\Quote\Api\Data\TotalsInterface;
+use Magento\Quote\Api\PaymentMethodManagementInterface;
+use Magento\Quote\Model\Quote;
+use Magento\Quote\Model\Quote\Address;
+use Magento\Quote\Model\QuoteAddressValidator;
+use Magento\Quote\Model\Shipping;
+use Magento\Quote\Model\ShippingAssignment;
+use Magento\Quote\Model\ShippingAssignmentFactory;
+use Magento\Quote\Model\ShippingFactory;
+use PHPUnit\Framework\MockObject\MockObject as MockObject;
+use PHPUnit\Framework\TestCase;
+
 /**
+ * Test for \Magento\Checkout\Model\ShippingInformationManagement.
+ *
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  * @SuppressWarnings(PHPMD.TooManyFields)
  */
-class ShippingInformationManagementTest extends \PHPUnit\Framework\TestCase
+class ShippingInformationManagementTest extends TestCase
 {
     /**
-     * @var \Magento\Framework\TestFramework\Unit\Helper\ObjectManager
+     * Stub cart id
+     *
+     * @var int
      */
-    private $objectManager;
+    private const STUB_CART_ID = 100;
+
+    /**
+     * Stub items count
+     *
+     * @var int
+     */
+    private const STUB_ITEMS_COUNT = 99;
```

Comment:
You don't need to provide DocBlocks for `const`. Even more - `const` is not `@var` :-(